.github/workflows/deploy.yml

```diff
@@ -12,98 +12,50 @@ on:
-            touch -d @$(( d + i )) "$file"
-          done
-
-      - name: Install Cloud SDK
-        uses: google-github-actions/setup-gcloud@v0
-        with:
-          version: 367.0.0
-          service_account_key: ${{ secrets.GKE_SA_KEY }}
-          project_id: ${{ secrets.GKE_PROJECT }}
-      
-      - name: Configure Docker
-        run: gcloud auth configure-docker $CONTAINER_REGISTRY
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
 
-      - name: Build and push api
-        uses: docker/build-push-action@v2
-        with:
-          context: ./
-          file: ./cmd/api/Dockerfile
-          builder: ${{ steps.buildx.outputs.name }}
-          push: true
-          tags: ${{ env.CONTAINER_REGISTRY }}/${{ secrets.GKE_PROJECT }}/textile/tableland/api:sha-${{ steps.prep.outputs.sha_short }}
-          cache-from: type=local,src=/tmp/.buildx-cache
-          cache-to: type=local,dest=/tmp/.buildx-cache
-      - name: Build and push healthbot
-        uses: docker/build-push-action@v2
+      - name: Connect via SSH, update the repo and run
+        uses: appleboy/ssh-action@master
         with:
-          context: ./
-          file: ./cmd/healthbot/Dockerfile
-          builder: ${{ steps.buildx.outputs.name }}
-          push: true
-          tags: ${{ env.CONTAINER_REGISTRY }}/${{ secrets.GKE_PROJECT }}/textile/tableland/healthbot:sha-${{ steps.prep.outputs.sha_short }}
-          cache-from: type=local,src=/tmp/.buildx-cache
-          cache-to: type=local,dest=/tmp/.buildx-cache
+          host: ${{ steps.prep.outputs.vm_host }}
+          username: validator
+          key: ${{ env.vm_ssh_key }}
+          port: 22
+          script_stop: true
+          script: |
+            cd go-tableland
+            git fetch && git checkout -f ${{ github.sha }}
+            cd docker
+            HEALTHBOT_ENABLED=true make ${{ steps.prep.outputs.deployment }}-up
+
+            # Wait to start and double-check we're running the expected version.
+            # This helps having a green check in the GH Action be high-signal that everything is fine.
+            sleep 5;
+            for i in 1 2 3 4; do [ $(curl --insecure --silent https://127.0.0.1/version | jq .GitCommit) = "\"${{ steps.prep.outputs.sha_short}}\"" ] && break || (if [ $i = 4 ]; then exit -1; else sleep 5; fi ) done 
+            echo "All healthy!"
 
-      - name: Deploy to GKE
-        working-directory: ./k8
-        run: |
-          gcloud container clusters get-credentials $GKE_CLUSTER --region $GKE_REGION --project ${{ secrets.GKE_PROJECT }}
-          export API_IMAGE=$CONTAINER_REGISTRY/${{ secrets.GKE_PROJECT }}/textile/tableland/api:sha-${{ steps.prep.outputs.sha_short }}
-          export HEALTHBOT_IMAGE=$CONTAINER_REGISTRY/${{ secrets.GKE_PROJECT }}/textile/tableland/healthbot:sha-${{ steps.prep.outputs.sha_short }}
-          DEPLOYMENT=${{ steps.prep.outputs.deployment }} make deploy
```

Comment:
As a last comment about this action; I think we should eventually still create _public_ Docker images.
Maybe only in creating releases in the repo or similar; I think some people would value that.